pom.xml

```diff
@@ -29,7 +29,7 @@
 
   <groupId>org.terracotta.internal</groupId>
   <artifactId>terracotta-apis-root</artifactId>
-  <version>1.8-SNAPSHOT</version>
+  <version>1.9-SNAPSHOT</version>
```

Comment:
set the new version using `mvn versions:set -DnewVersion=...` which will set the new version in all the pots

Reply:
My bad... incremented the version last minute and did it in too much of a hurry.